docs/doc/01-cloud/20-bi-tool/00-metabase.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Connect Databend Cloud To Metabase
+sidebar_label: Metabase
+description: 
+  Connect Databend Cloud To Metabase
+---
+
+[Metabase](https://www.metabase.com/) is a simple and powerful analytics tool which lets anyone learn and make decisions from their company’s data—no technical knowledge required.
+
+If you already sign in to the [Databend Cloud](../../10-deploy/20-databend-cloud.md) let's begin to connect Databend Cloud to Metabase.
+
+## Install Metabase
+
+Metabase is a Java application that can be executed by simply downloading the [JAR file](https://www.metabase.com/start/oss/jar). This is the only command you’ll need:
+
+```shell
+java -jar metabase.jar
+```
+
+This will launch a Metabase server on port 3000 by default.
+
+Metabase requires that you have Java 8 or higher available on your system.
+
+## Download the ClickHouse plugin for Metabase
+
+Because Databend is compatible with the ClickHouse protocol, we can use ClickHouse plugin to help Databend connect to Metabase.
+
+1. In general, the folder named `plugins` and metabase.jar are in the same folder. But if you do not have a plugins folder, create one as a subfolder of where you have metabase.jar saved.
+
+2. The ClickHouse metabase plugin is a JAR file named clickhouse.metabase-driver.jar. Download the latest version of the [JAR file](https://github.com/enqueue/metabase-clickhouse-driver/releases/latest).
+
+3. Save clickhouse.metabase-driver.jar in metabase plugins folder.
```

Comment:
I guess this step can combine with the last step. For example,

Download the latest version of the ClickHouse Metabase plugin to the folder you created in step 1. The plugin comes with a JAR file called xxxx.